prometheus_hardware_exporter/exporter.py

```diff
@@ -20,7 +20,7 @@ class ThreadingWSGIServer(ThreadingMixIn, WSGIServer):
 
 
 class SlientRequestHandler(WSGIRequestHandler):
```

Comment:
**nitpick:** 

The class name is SlientRequestHandler

Reply:
Done